### ISSUE TITLE
DPI COMPAT - methods duplicate (with different datatype) 

### ISSUE THREAD

**Issue by svenka3** [AsFigo/pyslint#144]
```verilog

import "DPI-C" function shortint my_and5 (logic [3:0] a, logic [3:0] b);

// VCS warns that this is the same name used for the earlier declaration but
// doesn't cause an error.
import "DPI-C" function int my_and5 (logic [3:0] a, logic [3:0] b);


module m;
endmodule : m
```

Add COMPAT check
